src/input_cp2k_dft.F

```diff
@@ -225,9 +225,12 @@ SUBROUTINE create_dft_section(section)
                           "towards a spin-polarized solution. "// &
                           "Thus, larger tolerance increases chances of ending up "// &
                           "with spin-polarization. "// &
-                          "More details in a GitHub `thread <https://github.com/cp2k/cp2k/issues/4389>`. "// &
                           "This option is only valid for unrestricted (i.e. spin polarised) "// &
-                          "Kohn-Sham (UKS) calculations.", &
+                          "Kohn-Sham (UKS) calculations. It also needs non-zero "// &
+                          "[ADDED_MOS](#CP2K_INPUT.FORCE_EVAL.DFT.SCF.ADDED_MOS) to actually affect the calculations, "// &
+                          "which is why it is not expected to work with [OT](#CP2K_INPUT.FORCE_EVAL.DFT.SCF.OT) "// &
+                          "and may raise errors when used with OT."// &
```

Comment:
The precommit check fails because there is a space missing:
```suggestion
                          "and may raise errors when used with OT. "// &
```